boards/arm/stm32/nucleo-g431rb/scripts/ld.script

```diff
@@ -47,7 +47,7 @@
 
 MEMORY
 {
-    flash (rx) : ORIGIN = 0x08000000, LENGTH = 128K
+    flash (rx) : ORIGIN = 0x08000000, LENGTH = 256K
```

Comment:
Why has the STM32 flash memory size been increased?

![Image](https://github.com/user-attachments/assets/569da44b-b968-4c6b-9611-500fac860c70)